### PR TITLE
feat: release version 2.6.1

### PR DESCRIPTION
**Pull request by nic-chen**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

release version 2.6.1

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [ ] Have you added corresponding test cases?
- [x] Have you modified the corresponding document?
- [x] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

**Comment by bisakhmondal** [comment]
@nic-chen, could we merge #1885 into the release build? It'd be helpful for end-users.

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,6 +31,39 @@
 - [1.5.0](#150)
 - [1.0.0](#100)
 
+# 2.6.1
+
+This release mainly bugfix and improves test cases for `Manager API`.
```
Comment (tokers) [inline CHANGELOG.md:36]:
```suggestion
This release mainly contains several bugfixes and test case improvements for `Manager API`.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -30,6 +31,39 @@
...
+
+`Manager API` 2.6.1 should be used with [Apache APISIX 2.6](https://apisix.apache.org/downloads/). It is not recommended to use with other Apache APISIX versions.
```
Comment (tokers) [inline CHANGELOG.md:38]:
```suggestion
Note `Manager API` `2.6.1` should be used with [Apache APISIX 2.6](https://apisix.apache.org/downloads/). It is not recommended to use it with other Apache APISIX versions.
```

File: CHANGELOG.md (hunk 1 above)
Comment (tokers) [reply CHANGELOG.md:36]: By the way, this sentence is not correct as there are some features listed in the change log context.

File: CHANGELOG.md [hunk 3]
```diff
@@ -30,6 +31,39 @@
...
+
+### Core
+
+- Feat: support HTTPS for Manager API [#1824](https://github.com/apache/apisix-dashboard/pull/1824)
+- Feat: run manager-api as an OS agnostic service [#1667](https://github.com/apache/apisix-dashboard/pull/1667)
+- Feat: refactor Plugin Orchestration [#1813](https://github.com/apache/apisix-dashboard/pull/1813)
+- Feat: add the service page upstream select option [#1633](https://github.com/apache/apisix-dashboard/pull/1633)
+- Feat: improve the Duplicate Route feature [#1833](https://github.com/apache/apisix-dashboard/pull/1833)
```
Comment (tokers) [inline CHANGELOG.md:46]: Actually, it's not a good practice when you release a patched version but carrying so many features.

File: api/internal/utils/consts/versionMap.go [hunk 4]
```diff
@@ -21,8 +21,9 @@ package consts
...
 var VersionMap = map[string]string{
-	"2.3": "2.2",
-	"2.4": "2.3",
-	"2.5": "2.4",
-	"2.6": "2.5",
+	"2.3":   "2.2",
+	"2.4":   "2.3",
+	"2.5":   "2.4",
+	"2.6":   "2.5",
+	"2.6.1": "2.6",
```
Comment (starsz) [inline api/internal/utils/consts/versionMap.go:28]: So the 2.6.1 should be used with 2.6?

File: CHANGELOG.md [hunk 5]
```diff
@@ -30,6 +31,39 @@
...
 
+# 2.6.1
+
+This release mainly bugfix and improves test cases for `Manager API`.
+
+`Manager API` 2.6.1 should be used with [Apache APISIX 2.6](https://apisix.apache.org/downloads/). It is not recommended to use with other Apache APISIX versions.
+
+### Core
+
+- Feat: support HTTPS for Manager API [#1824](https://github.com/apache/apisix-dashboard/pull/1824)
```
Comment (juzhiyuan) [inline CHANGELOG.md:42]: So many features for a patched version 🤣🤣

**Review by iamayushdas — APPROVED** [review]
LGTM, but there are so many features lying in this patch update

**Comment by imjoey** [comment]
@nic-chen I was wondering that if this `2.6.1` release is a patched version for `2.6.x`, cause there are several new features in it which is more likely a minor version update. Or is this a brand new version-match rule that the dashboard will be versioned as `x.y.z` for the corresponding apisix versions `x.y`, such as dashboard `2.6.1` with apisix `2.6`?

If this is a critical patched version, it would be better to only contain fixes for critical or serious bugs. After referring to the changelogs, there seems no such critical or serious ones. So I guess this may be a regular patched version upgrade, does it mean we are going to maintain multiple versions? 

Looking forward to your response, thank you very much.

**Comment by nic-chen** [comment]
OK.

A patched version that only contains bugfixes is more reasonable. 

This pr will be closed and the release process will be restarted.

**Pull request closed without merging**